$OpenBSD: patch-Makefile_PL,v 1.2 2017/06/01 12:18:54 espie Exp $

https://rt.cpan.org/Public/Bug/Display.html?id=117793

Also, add a zero to defined constants. So if the define is empty, the code
still makes sense.

Index: Makefile.PL
--- Makefile.PL.orig
+++ Makefile.PL
@@ -127,7 +127,7 @@ if (!defined($curl_h)) {
     close H;
 
     for my $e (sort @syms) {
-       if($e =~ /(OBSOLETE|^CURL_EXTERN|_LAST\z|_LASTENTRY\z)/) {
+       if($e =~ /(OBSOLETE|^CURL_EXTERN|^CURL_STRICTER\z|_LAST\z|_LASTENTRY\z)/) {
           next;
        }
        my ($group) = $e =~ m/^([^_]+_)/;
@@ -190,7 +190,7 @@ HERE2
             if ($next_initial eq $initial) {
 
                 print CURL_XS
-"            if (strEQ(name, \"$remainder\")) return "."$groupref->{$option};\n";
+"            if (strEQ(name, \"$remainder\")) return "."$groupref->{$option}+0;\n";
 
                 $count++;
             }
